games-engines/scummvm/scummvm-1.8.1.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="Script Creation Utility for Maniac Mansion Virtual Machine"
+DESCRIPTION="ScummVM is a program which allows you to run certain classic \
+graphical point-and-click adventure games, provided you already have their \
+data files. The clever part about this: ScummVM just replaces the executables \
+shipped with the games, allowing you to play them on systems for which they \
+were never designed!"
+HOMEPAGE="http://scummvm.org"
+COPYRIGHT="2001-2014 ScummVM Team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://scummvm.org/frs/scummvm/$portVersion/scummvm-$portVersion.tar.xz"
+CHECKSUM_SHA256="935f53d7e4c460fbc22c58fee5007d4858891e4961831f53f575a3594b7f612a"
+ADDITIONAL_FILES="scummvm.rdef"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	scummvm$secondaryArchSuffix = $portVersion
+	app:scummvm$secondaryArchSuffix = $portVersion
+"
```

Comment:
We like to add a tab before the closing double quotes, like this:
```Shell
PROVIDES="
	scummvm$secondaryArchSuffix = $portVersion
	app:scummvm$secondaryArchSuffix = $portVersion
	"
```

Of couse, the same applies to REQUIRES, BUILD_REQUIRES and BUILD_PREREQUIRES.